scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -3,14 +3,16 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
 	var pop_before=0,pop_after=0,reduced_bombard_score=0,strength_reduction=0,txt2="",txt3="",txt4="",max_kill,overkill,roll,kill;
 	var score_before=star_system.p_population[planet_number];
 
-	var txt1="The heavens rumble and thunder as your ship";
+	// TODO - update descriptions below, once we get Surface to Orbit weaponry into the game
+
+	var txt1="Your cruiser and larger ship"; // TODO - add more variation, for different planets, perhaps different ships, CMs positioning, planetary features and other factors
 	if (ships_selected>1) then txt1+="s";
-	txt1+=" unload";
+	txt1+=" position themselves over the target in close orbit, and unleash";
 	if (ships_selected=1) then txt1+="s";
-	txt1+= $" annihilation upon {planet_numeral_name(planet_number, star_system)}.  Even from space the explosions can be seen, clapping across the planet's surface.";
+	txt1+= $" annihilation upon {planet_numeral_name(planet_number, star_system)}.  Even from space the explosions can be seen , {choose("tearing ground", "hammering", "battering", "thundering")} across the planet's surface.";
 
 	if (star_system.p_large[planet_number]=0){
-		kill=bombard_ment_power*15000000;// Population if normal
+		kill=bombard_ment_power*15000000;// Population if normal, TODO consider making loses to be more percentage-wise, rather than flat, also in scr_purge_world
 	}else if (star_system.p_large[planet_number]=1){
 		kill=bombard_ment_power*0.15;// Population if large
 	}
```

Comment:
```suggestion
	var pop_before=0,pop_after=0,reduced_bombard_score=0,strength_reduction=0,txt2="",txt3="",txt4="",max_kill,overkill,roll,kill;
	var score_before=star_system.p_population[planet_number];

	// TODO - update descriptions below, once we get Surface to Orbit weaponry into the game

	var txt1=choose("Your cruiser and larger ship", "The heavens rumble and thunder as your ship"); // TODO - add more variation, for different planets, perhaps different ships, CMs positioning, planetary features and other factors
	if (ships_selected>1) then txt1+="s";
	txt1+=choose(" position themselves over the target in close orbit, and unleash", " unload");
	if (ships_selected=1) then txt1+="s";
	txt1+= $" annihilation upon {planet_numeral_name(planet_number, star_system)}. Even from space the explosions can be seen, {choose("tearing ground", "hammering", "battering", "thundering")} across the planet's surface.";

	if (star_system.p_large[planet_number]=0){
		kill=bombard_ment_power*15000000;// Population if normal, TODO consider making loses to be more percentage-wise, rather than flat, also in scr_purge_world
	}else if (star_system.p_large[planet_number]=1){
		kill=bombard_ment_power*0.15;// Population if large
	}
```